recipes/pyspatialstats/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "pyspatialstats" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/pyspatialstats-{{ version }}.tar.gz
+  sha256: 0d15361f98bd15acf275404c960be9140977ab3ea993dee7923660a5087ebc64
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-build-isolation
+  number: 0
+  skip: true  # [py<310 or python_impl=="pypy"]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ stdlib("c") }}
+    - numpy >=2.0
+  host:
+    - python
+    - setuptools
+    - wheel
+    - numpy >=2.0
+    - cython >=3.0
+    - pip
+  run:
+    - python
+    - numpydantic
+    - joblib
+    - pandas
+    - scipy
+    - {{ pin_compatible('numpy') }}
```

We no longer need this for numpy 2.0

```suggestion
    - numpy
```